quesma/quesma/config/config_v2.go

```diff
@@ -603,11 +609,38 @@ func (c *QuesmaNewConfiguration) TranslateToLegacyConfig() QuesmaConfiguration {
 		}
 
 		conf.IndexConfig = make(map[string]IndexConfiguration)
+
+		// Handle default index configuration
+		defaultConfig := queryProcessor.Config.IndexConfig[DefaultWildcardIndexName]
+		for _, target := range defaultConfig.Target {
```

Comment:
Just for consistency, maybe better would be to change this for loop to be same as below one with the exception of using `queryProcessor` or change name from `defaultConfig`  to `queryDefaultConfig` to be more meaningful.

Potentially, we could also try to extract a common function

Reply:
> change name from defaultConfig to queryDefaultConfig

That's not possible - both loops share the same `defaultConfig`, one fills out `QueryTarget` of it, while the second one fills out `IngestTarget` of it.

> Potentially, we could also try to extract a common function

Agreed, I'll do it in the follow up PR once we have integration tests covering this case.